x/evm/types/msg.go

```diff
@@ -37,7 +37,6 @@ type (
 		Data TxData
 
 		// caches
-		hash atomic.Value
```

Comment:
what's the lint error here? afaik this should be used

Reply:
cc @noot ^^

Reply:
```
x/evm/types/msg.go:40:3: `hash` is unused (structcheck)
                hash atomic.Value
```

Reply:
not sure about this one, I didn't write this part. i think since Austin removed the hash function for this type it's unused now?

Reply:
we can always revert it back if something breaks.

Reply:
yeah there is no need for it currently because you need to know the tx encoding protocol to hash, so this hash isn't cached, but it can definitely be changed to do so (might be worth it later)

Reply:
my bad I didn't think to remove this, was just a quick fix before, but can add a utility function to hash only if cached hash not set (I indicated this as being ideal, but not a priority, but might be good to at least leave a TODO)